firmware/src/magnet/charger.cpp

```diff
@@ -160,14 +157,14 @@ Charger::Status Charger::runAndGetStatus()
     }
 
     // Print supply Voltage when below 4.8V
-    unsigned const Vout = board::getOutVoltageInVolts();
-    if (Vout >= target_output_voltage_ && supply_volatage_mV_min <= 4800)
+    unsigned const OutputVoltage = board::getOutVoltageInVolts();
+    if (OutputVoltage >= target_output_voltage_ && supply_volatage_mV_min <= 4800)
```

Comment:
`OutputVoltage` is not a static const, nor a constexpr, so it should be in `snake_case`. Also, `const` goes first, then type name (it's a default convention everywhere).